sites/svelte-5-preview/src/routes/docs/content/01-api/02-runes.md

```diff
@@ -250,40 +250,34 @@ $effect(() => {
 });
 ```
 
-An effect only reruns when the object it reads changes, not when a property inside it changes. If you want to react to _any_ change inside an object for inspection purposes at dev time, you may want to use [`inspect`](#$inspect).
+An effect only reruns when the object it reads changes, not when a property inside it changes. (If you want to observe changes _inside_ an object at dev time, you can use [`$inspect`](#$inspect).)
 
 ```svelte
 <script>
-	let object = $state({ count: 0 });
-	let derived_object = $derived({
-		doubled: object.count * 2
-	});
+	let a = $state({ value: 0 });
+	let b = $derived({ value: a.value * 2 });
```

Comment:
These should have more descriptive names, `a` and `b` make it harder to understand what I'm looking at further below as a beginner.

Reply:
I don't buy this — the whole thing fits comfortably in a single screen, even on a normal mobile phone. Abstract things can have abstract names

Reply:
My eyes still have to jump back and forth more when I read `a` or `b` compared to something like `count` and `double` or `state` and `derived`. `a` and `b` are just way too generic

Reply:
can go with `state = $state(...)` and `derived = $derived(...)` if you prefer that. `count` and `double` don't say anything more useful than `a` and `b` (since the values they hold aren't the point of this example)

Reply:
Let's go with `state` and `derived` then. (I think `count` and `double` are still better than `a` and `b` because they are easier to memorize)